features/add_to_cart.feature

```diff
@@ -19,19 +19,21 @@ Feature: user can can add dish to order
 
   Scenario: User can add dish to order
     Then I should see the text "Sushi rolls"
+    And show me the page
```

Comment:
please remove

Reply:
One show me the page besides that good job (y)

Reply:
It was already gone :D